src/main/java/screens/CardScreen.java

```diff
@@ -6,15 +6,11 @@
 
 import javax.swing.*;
 import java.awt.*;
-import java.awt.event.ActionEvent;
-import java.awt.event.ActionListener;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Objects;
 
-import static javax.swing.JOptionPane.showMessageDialog;
-
 public class CardScreen extends JPanel {
```

Comment:
if you've got time maybe comment on what's going on?

Reply:
also javadoc

Reply:
Removed unnecessary import statements. We did not end up using that